packages/integration-tests/utils.js

```diff
@@ -3,6 +3,22 @@
 const { readJsonFile } = require('@cumulus/common/FileUtils');
 const path = require('path');
 const fs = require('fs-extra');
+/**
+* Returns string appended with _[0-9,a-z] up to 35 batches)
+* @summary If the description is long, write your summary here. Otherwise, feel free to remove this.
```

Comment:
Leftover summary? Definitely an 🐜

Reply:
Yikes, forgot to fill in the jsdoc template.   Lorem Ipsum, etc.     Thanks for noting it!

Reply:
777406a7f